src/components/HomePage/HomeHero.tsx

```diff
@@ -0,0 +1,246 @@
+    const x2 = transform(mX, [-1, 1], [LINE_MIN_APEX_X, LINE_MAX_APEX_X]);
+    const x3 = transform(
+      mX,
+      [-1, 1],
+      [LINE_MAX_APEX_X - SEGMENT_WIDTH, LINE_WIDTH],
+    );
+    const x = (v: number): number => Math.max(0, Math.min(LINE_WIDTH, v));
+
+    const yMod =
+      mY >= 0
+        ? transform(index, LINE_RANGE, [0, 1])
+        : transform(index, LINE_RANGE, [1, 0]);
+    const peakY = LINE_APEX_Y * mY * yMod;
+    const yValue = transform(index, LINE_RANGE, [0, LINE_WIDTH - LINE_HEIGHT]);
+    const y = (v: number): number => v + yValue;
+
+    return [
+      `M0,${y(0)}`,
+      `L${x(x1)},${y(0)}`,
+      // prettier-ignore
+      `C${x(x1 + SEGMENT_WIDTH)},${y(0)} ${x(x2 - SEGMENT_WIDTH)},${y(peakY)} ${x(x2)},${y(peakY)}`,
+      // prettier-ignore
+      `C${x(x2 + SEGMENT_WIDTH)},${y(peakY)} ${x(x3 - SEGMENT_WIDTH)},${y(0)} ${x(x3)},${y(0)}`,
+      `L1000,${y(0)}`,
+      `L1000,${y(LINE_HEIGHT)}`,
+      `L${x(x3)},${y(LINE_HEIGHT)}`,
+      // prettier-ignore
+      `C${x(x3 - SEGMENT_WIDTH)},${y(LINE_HEIGHT)} ${x(x2 + SEGMENT_WIDTH)},${y(peakY + LINE_HEIGHT)} ${x(x2)},${y(peakY + LINE_HEIGHT)}`,
+      // prettier-ignore
+      `C${x(x2 - SEGMENT_WIDTH)},${y(peakY + LINE_HEIGHT)} ${x(x1 + SEGMENT_WIDTH)},${y(LINE_HEIGHT)} ${x(x1)},${y(LINE_HEIGHT)}`,
+      `L0,${y(LINE_HEIGHT)}`,
+      `Z`,
+    ].join(' ');
+  });
+
+  const opacity = useTransform(mouse, ([, mY]) => {
+    const indexMod =
+      mY >= 0
+        ? transform(index, LINE_RANGE, [0, 1])
+        : transform(index, LINE_RANGE, [1, 0]);
+
+    return transform(Math.abs(mY * indexMod), [0, 1], [LINE_MIN_OPACITY, 1]);
+  });
+
+  return (
+    <motion.path
+      d={d}
+      fill="url('#hh_lg_0')"
+      stroke="transparent"
+      strokeWidth="0.0125rem"
+      style={{ opacity }}
+      width="100%"
+    />
+  );
+};
+
+const STOP_COLORS = interpolateColors(
+  [theme.colors.midBlue, theme.colors.coral, theme.colors.midBlue],
+  4,
+);
+
+const GRADIENT_CENTER_OFFSET = LINE_MIN_APEX_X / LINE_WIDTH;
+
+export const HomeHero: FC = () => {
+  const [ref, { point }] = useMouseAnimation<HTMLDivElement>({
+    damping: 10,
+    stiffness: 75,
+  });
+  const x1 = useTransform(point, ([mX]) => {
+    return transform(
+      mX,
+      [-1, 1],
+      [-GRADIENT_CENTER_OFFSET, GRADIENT_CENTER_OFFSET],
+    );
+  });
+  const x2 = useTransform(point, ([mX]) => {
+    return transform(
+      mX,
+      [-1, 1],
+      [1 - GRADIENT_CENTER_OFFSET, 1 + GRADIENT_CENTER_OFFSET],
+    );
+  });
+
+  return (
+    <div
+      css={css`
+        display: grid;
+        gap: ${cssClamp([2, 'xs'], [1.5, 'lg'])};
+        padding: ${cssClamp([3, 'xs'], [10.5, 'lg'])}
+          ${cssClamp([1.5, 'xs'], [5.25, 'lg'])};
+        position: relative;
+        text-align: center;
+
+        & * {
+          text-shadow: 0.25rem 0.25rem 0.375rem ${theme.colors.darkBlue};
+          z-index: 1;
+        }
+
+        & svg {
+          z-index: 0;
```

Comment:
💬 I think that this would look better stretched edge to edge. When I slapped the following into Chrome dev tools it seemed to work, but it may not be the "right way" to do it.

```suggestion
          left: calc(50% - 50vw);
          width: 100vw;          
          z-index: 0;
```

Reply:
🏆 Excellent suggestion. I went ahead and incorporated that and pushed.